clouds/microsoft_azure/utils.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+from simple_logger.logger import get_logger
+from azure.mgmt.subscription import SubscriptionClient
+from azure.mgmt.redhatopenshift import AzureRedHatOpenShiftClient
+
+
+LOGGER = get_logger(name="microsoft_azure-utils")
```

Comment:
please keep the same format in the name; either use "-" or "_"

Reply:
done